frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -1,6 +1,15 @@
 <template>
   <div>
-    <h1 class="card-title">{{ title }}</h1>
+    <div class="row">
+      <div class="col col-md-4">
```

Comment:
```suggestion
      <div class="col col-md-4 d-flex align-items-center"">
```

se puede centrar verticalmente esto? me da un algo que no se ve centrado, jaja.